### PR TITLE
OSD-5518 allow dedicated-admins on hive (int) to manage velero schedules

### PR DESCRIPTION
**Pull request by yithian**

### CODE REVIEW

**Comment by yithian** [comment]
I have no idea what these other diffs are about. They appear when `make` is run on a clean checkout of the master branch.

```diff
-          namespacesAllowedRegex: .*
-          namespacesDeniedRegex: (^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)
+          namespacesAllowedRegex: (^redhat-rhmi-.*|^redhat-rhoam-.*)
```

File: hack/00-osd-managed-cluster-config-integration.yaml.tmpl [hunk 1]
```diff
@@ -1052,8 +1052,7 @@ objects:
         permissions:
         - allowFirst: true
           clusterRoleName: backplane-cssre-admins-project
-          namespacesAllowedRegex: .*
-          namespacesDeniedRegex: (^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)
+          namespacesAllowedRegex: (^redhat-rhmi-.*|^redhat-rhoam-.*)
```
Comment (cblecker) [inline hack/00-osd-managed-cluster-config-integration.yaml.tmpl:1055]: this seems wrong

Comment (cblecker) [reply hack/00-osd-managed-cluster-config-integration.yaml.tmpl:1055]: Oh. This is from #545

Comment (yithian) [reply hack/00-osd-managed-cluster-config-integration.yaml.tmpl:1055]: looks like this got fixed on master. rebasing and force-pushing a branch with just my changes (ie: excluding eb6a7c7)

**Pull request closed without merging**